build.sbt

```diff
@@ -5,5 +5,6 @@ version := "0.1"
 scalaVersion := "2.12.3"
 
 libraryDependencies ++= Seq(
-  "org.typelevel" %% "spire" % "0.14.1"
-)
\ No newline at end of file
+  "org.typelevel" %% "spire" % "0.14.1",
+  "org.scalatest" %% "scalatest" % "3.0.0"
```

Comment:
Should this be in a different SBT scope?

Reply:
wups totally right